### PR TITLE
Sim gui config

### PR DESCRIPTION
**Pull request by LucienMorey**
Add a basic sim gui config to save a bit of time and effort when trying to run the sim. Isee these files are in the gitignore and am not sure yet if this will cause grief. I am have set the ```skip-worktree``` flag which I am hoping will mean that this configuration is respected until we decide to change it again

### CODE REVIEW

**Comment by auscompgeek** [comment]
> I am have set the skip-worktree flag

Those flags don't get committed, they only apply to your local index.

File: simgui.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            0.0,
+            1.0,
+            255.0
+          ]
+        },
+        "height": 8.013679504394531,
+        "left": 46,
+        "right": 1088,
+        "top": 36,
+        "width": 16.541748046875,
```
Comment (auscompgeek) [inline simgui.json:24]: These field dimensions are wrong, notice the yellow border doesn't line up with the walls in the field image:

screenshot

**Review by auscompgeek — CHANGES_REQUESTED** [review]

File: simgui.json (hunk 1 above)
Comment (james-ward) [reply simgui.json:24]: I have a patch that uses the newest image from WPILib, and sets the coordinates correctly.
The `.png` file is 2MB. Should I force commit it and add to this PR?

Comment (james-ward) [reply simgui.json]: I have pushed the correct dimensions

**Comment by james-ward** [comment]
I have changed the approach - the three files are in the `sim` subdirectory and they have an added `.orig` extension.
To use them they are copied into the root directory and `.orig` stripped off. This way `git` won't try to track these files each time they are modified by the sim.

File: sim/simgui-window.json.orig [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (auscompgeek) [inline sim/simgui-window.json.orig:1]: Do we have a good reason to check in a window file at all? The WPILib template specifically ignores all the window files. https://github.com/wpilibsuite/vscode-wpilib/blob/ccc008aef0ad04ddcd2595edb4d982250504df30/vscode-wpilib/resources/gradle/shared/.gitignore#L172

Comment (james-ward) [reply sim/simgui-window.json.orig:1]: Let's remove it! Problem solved!

**Review by james-ward — APPROVED** [review]

**Pull request merged**